tailwind.config.ts

```diff
@@ -0,0 +1,54 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
+  ],
+  theme: {
+    fontSize: {
+      xs: "0.75rem",
+      sm: "0.875rem",
+      base: "1rem",
+      lg: "1.125rem",
+      xl: "1.25rem",
+      "2xl": "1.5rem",
+      "3xl": "1.875rem",
+      "4xl": "2.25rem",
+      "5xl": "3rem",
+      "6xl": "4rem",
+    },
+    extend: {
+      colors: {
+        primary: {
+          100: "#E6F6FE",
+          200: "#C0EAFC",
+          300: "#9ADDFB",
+          400: "#4FC3F7",
+          500: "#03A9F4",
+          600: "#0398DC",
+          700: "#026592",
+          800: "#014C6E",
+          900: "#013349",
+        },
+        gray: {
+          100: "#f7fafc",
+          200: "#edf2f7",
+          300: "#e2e8f0",
+          400: "#cbd5e0",
+          500: "#a0aec0",
+          600: "#718096",
+          700: "#4a5568",
```

Comment:
We shouldn't include any custom colors yet since we haven't come up with the color palette for the UI.

Reply:
I will have it there for now as its not used. Just more convenient in the future we have a pallette, then just simply replace it with HEX code from Figma file for examples